lib/src/client.dart

```diff
@@ -121,10 +120,7 @@ class Client {
           'parameters, was "$parameters".');
     }
 
-    var message = <String, dynamic>{
-      "jsonrpc": "2.0",
-      "method": method
-    };
+    var message = <String, dynamic>{"jsonrpc": "2.0", "method": method};
```

Comment:
Please remove these incidental reformattings. This package doesn't use `dartfmt`, and they make it harder to see the real changes.